docker-compose.yml

```diff
@@ -338,7 +339,7 @@ services:
       - CLEANUP_TASK_AND_BEATTRFORM=FALSE
       - FULL_MYSQL_URL=mysql/gennydb?serverTimezone=UTC
       # - KEYCLOAK_AUTH_URL=https://keycloak-office.gada.io/auth/realms/internmatch
-      - KEYCLOAK_AUTH_URL=https://keycloak.gada.io/auth:${GENNY_KEYCLOAK_URL}/realms/${GENNY_KEYCLOAK_REALM}
+      - KEYCLOAK_AUTH_URL=https://keycloak-bali.gada.io/realms/${GENNY_KEYCLOAK_REALM}
```

Comment:
instead of hard code keycloak url here, I suggest you use ${GENNY_KEYCLOAK_URL} like you already did 
`KEYCLOAKURL=${GENNY_KEYCLOAK_URL}/realms` is more flexible

Reply:
Fixed